examples/deployment/docker/log_server/Dockerfile

```diff
@@ -8,6 +8,12 @@ RUN go get ./server/trillian_log_server
 
 FROM gcr.io/distroless/base
 
-COPY --from=build /go/bin/trillian_log_server /
+# Include the source code for Trillian and all of its transitive dependencies.
+# This ensures that we meet their license requirements to redistribute their
+# licenses, copyright notices and, in some cases, source code.
+COPY --from=build \
+	/go/bin/trillian_log_server \
+	/go/src \
+	/
```

Comment:
What do we need from `/`?  If we have it, could we just drop the lines above?

Reply:
As discussed in person, "/" is the destination, not a source. I've split this into two COPY commands, which makes this more obvious.